raiden/network/transport/matrix/transport.py

```diff
@@ -73,6 +74,9 @@
 log = structlog.get_logger(__name__)
 
 _RoomID = NewType("_RoomID", str)
+# Combined with 10 retries (``..utils.JOIN_RETRIES``) this will give a total wait time of ~15s
+ROOM_JOIN_RETRY_INTERVAL = 0.1
+ROOM_JOIN_RETRY_INTERVAL_MULTIPLIER = 1.55
```

Comment:
If we define the back off here explicitly, can we also get rid of the three config values for the RetryQueue? 
For me it makes sense to have the constants close to the code that uses them instead of forwarding them from service to transport to queue, but whatever pattern we choose it would be nice if it is consistent.

Reply:
The config values are about message retry. 
These ones are only used for room invite joins